modules/codemod/lib/v7/recategorizeIconButtons.ts

```diff
@@ -73,7 +74,19 @@ export default function transformer(file: FileInfo, api: API, options: Options)
 
       // Default IconButton variant is `circle`
       if (variantProp) {
-        const variantPropValue = ((variantProp as JSXAttribute).value as StringLiteral)?.value;
```

Comment:
I know `variantProp as JSXAttribute` was present before, but with so many more usages I'm checking a way to remove the need to use `as` over and over. I'll let you know if I figure it out.